Unable to build after dep ensure

Seems like there's a major (breaking) change on `ulule/limiter` package

**Reproduction Steps:**

1. Pull `janus` repo
2. Make deps
3. Run binary

**Observed behavior:**
```
pkg/plugin/rate/rate_limit_logger.go:60:26: not enough arguments in call to lmt.Peek
	have (string)
	want (context.Context, string)
pkg/plugin/rate/setup.go:74:21: undefined: limiter.NewLimiter
pkg/plugin/rate/setup.go:76:19: undefined: limiter.NewHTTPMiddleware
pkg/plugin/rate/setup.go:94:10: undefined: limiter.NewRedisStoreWithOptions
pkg/plugin/rate/setup.go:99:10: undefined: limiter.NewMemoryStore
# github.com/hellofresh/janus/pkg/plugin/oauth2
pkg/plugin/oauth2/loader.go:52:19: undefined: limiter.NewMemoryStore
pkg/plugin/oauth2/loader.go:53:22: undefined: limiter.NewLimiter
pkg/plugin/oauth2/loader.go:54:22: undefined: limiter.NewHTTPMiddleware
```

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -104,7 +104,7 @@
   name = "github.com/tidwall/gjson"
 
 [[constraint]]
-  branch = "master"
+  branch = "1.0.0"
   name = "github.com/ulule/limiter"
 
 [[constraint]]

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -104,8 +104,8 @@
   name = "github.com/tidwall/gjson"
 
 [[constraint]]
-  branch = "1.0.0"
   name = "github.com/ulule/limiter"
+  version = "1.0.0"
 
 [[constraint]]
   name = "gopkg.in/mgo.v2"

